--- openttd-1.1.0.orig/src/table/settings.h	2011-02-13 21:52:33.000000000 +0100
+++ openttd-1.1.0/src/table/settings.h	2011-04-30 15:18:23.358000016 +0200
@@ -270,11 +270,11 @@
 	  SDTG_STR("savegame_format",  SLE_STRB, S, 0, _savegame_format,       NULL,    STR_NULL, NULL),
 	 SDTG_BOOL("rightclick_emulate",         S, 0, _rightclick_emulate,   false,    STR_NULL, NULL),
 #ifdef WITH_FREETYPE
-	  SDTG_STR("small_font",       SLE_STRB, S, 0, _freetype.small_font,   NULL,    STR_NULL, NULL),
-	  SDTG_STR("medium_font",      SLE_STRB, S, 0, _freetype.medium_font,  NULL,    STR_NULL, NULL),
-	  SDTG_STR("large_font",       SLE_STRB, S, 0, _freetype.large_font,   NULL,    STR_NULL, NULL),
-	  SDTG_VAR("small_size",       SLE_UINT, S, 0, _freetype.small_size,   8, 0, 72, 0, STR_NULL, NULL),
-	  SDTG_VAR("medium_size",      SLE_UINT, S, 0, _freetype.medium_size, 10, 0, 72, 0, STR_NULL, NULL),
+	  SDTG_STR("small_font",       SLE_STRB, S, 0, _freetype.small_font,   "DejaVu Sans",    STR_NULL, NULL),
+	  SDTG_STR("medium_font",      SLE_STRB, S, 0, _freetype.medium_font,  "DejaVu Sans",    STR_NULL, NULL),
+	  SDTG_STR("large_font",       SLE_STRB, S, 0, _freetype.large_font,   "DejaVu Sans",    STR_NULL, NULL),
+	  SDTG_VAR("small_size",       SLE_UINT, S, 0, _freetype.small_size,  12, 0, 72, 0, STR_NULL, NULL),
+	  SDTG_VAR("medium_size",      SLE_UINT, S, 0, _freetype.medium_size, 15, 0, 72, 0, STR_NULL, NULL),
 	  SDTG_VAR("large_size",       SLE_UINT, S, 0, _freetype.large_size,  16, 0, 72, 0, STR_NULL, NULL),
 	 SDTG_BOOL("small_aa",                   S, 0, _freetype.small_aa,    false,    STR_NULL, NULL),
 	 SDTG_BOOL("medium_aa",                  S, 0, _freetype.medium_aa,   false,    STR_NULL, NULL),
